.eslintignore

```diff
@@ -1,9 +1,3 @@
-/blueprints/*/files
-
-# unconventional js
-/blueprints/*/files/
-/vendor/
-
 # compiled output
 /dist/
 /tmp/
```

Comment:
why do we keep these? do we expect to have these folders in the top-level folder?

Reply:
fixed